docs/src/configuration/routes/https.md

```diff
@@ -81,11 +81,15 @@ The use cases for this configuration are few.
 More complex routing logic is also possible if the situation calls for it.
 
 {{< note >}}
-Let's Encrypt has a limit of 100 TLS certificates per environment.
-If you define both a `{default}` and `www.{default}` route for each domain you use, that will give you a limit of 50 domains.
-Adding more than that will result in a warning on deploy and some domains will not be issued a TLS certificate.
-If you need more than that, we recommend obtaining additional certificates or a wildcard certificate from another TLS provider.
-Alternatively, consider splitting your project up into multiple discrete Platform.sh projects.
+
+Let's Encrypt has a limit of 100 hostnames per certificate.
```

Comment:
could we use domains instead of hostnames please?
```suggestion
Let's Encrypt has a limit of 100 domains (to be more precise 100 SAN or Subject Alternative Names) per certificate.
```

Reply:
@pjcdawkins requested hostnames instead of domains :smile:

Reply:
because

1. A "domain" is already a defined thing you can add to your platform.sh project. Each domain can result in 0 or more names in the TLS certificate, not necessarily 1 name.
1. "Hostname" is the word Let's Encrypt uses on their page:

https://letsencrypt.org/docs/rate-limits/
> You can combine multiple hostnames into a single certificate, up to a limit of 100 Names per Certificate.

Reply:
Solving the thread without applying the suggestion since it makes sense to stick with Let's Encrypt terminology (even if domain would be more what the customer might expect).